src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -0,0 +1,167 @@
+{
+  "title": "FV3 Forecast config",
+  "description": "This document is to validate user-defined FV3 forecast config files",
+  "type": "object",
+  "properties": {
+    "platform": {
+      "description": "attributes of the platform",
+      "type": "object",
+      "properties": {
+        "WORKFLOW_MANAGER": {
+          "type": "string",
+          "enum": [
+            "rocoto",
+            "none"
+          ]
+        },
+        "NCORES_PER_NODE": {
+          "type": "number"
+        },
+        "SCHED": {
+          "type": "string",
+          "enum": [
+            "slurm",
+            "pbspro",
+            "lsf",
+            "lsfcray",
+            "none"
+          ]
+        },
+        "CCPA_OBS_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "MRMS_OBS_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "NDAS_OBS_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "METPLUS_PATH": {
+          "type": "string",
+          "format": "uri"
+        },
+        "MET_BIN_EXEC": {
+          "type": "string"
+        },
+        "MET_INSTALL_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "DOMAIN_PREGEN_BASEDIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "PARTITION_DEFAULT": {
+          "type": "string"
+        },
+        "QUEUE_DEFAULT": {
+          "type": "string"
+        },
+        "PARTITION_FCST": {
+          "type": "string"
+        },
+        "QUEUE_FCST": {
+          "type": "string"
+        },
+        "PARTITION_HPSS": {
+          "type": "string"
+        },
+        "QUEUE_HPSS": {
+          "type": "string"
+        }
+      }
+    },
+    "parameters": {
```

Comment:
I'm not sure what these two parameters are. They are both listed as `uri` but `length` and `grid` as properties don't map to anything I'm particularly aware of.

Reply:
ah uri is a slip up from me, they should be number

Reply:
And the Jira issue requested "ex: needs to know node size, queues, length, grid, suites, locations", so I included the forecast length and grid here